src/background.ts

```diff
@@ -611,26 +611,102 @@ ipcMainHandle("GET_ALT_PORT_INFOS", () => {
   return engineManager.altPortInfo;
 });
 
+/**
+ * 保存に適したフォルダを選択するかキャンセルをするまでダイアログを繰り返し表示し続ける
```

Comment:
なぜ保存に適してないかわからないかも。ちょっと改変してみました。（フォルダかファイルかわからない）
```suggestion
 * 保存に適した場所を選択するかキャンセルするまでダイアログを繰り返し表示する。
 * アンインストール等で消えうる場所などを避ける。
```